gateway-js/src/index.ts

```diff
@@ -425,135 +425,153 @@ export class ApolloGateway implements GraphQLService {
     };
   }
 
-  // Synchronously load a statically configured schema, update class instance's
-  // schema and query planner.
-  private loadStatic(config: StaticGatewayConfig) {
-    let schema: GraphQLSchema;
-    let supergraphSdl: string;
-    try {
-      ({ schema, supergraphSdl } = isLocalConfig(config)
-        ? this.createSchemaFromServiceList(config.localServiceList)
-        : this.createSchemaFromSupergraphSdl(config.supergraphSdl));
-      // TODO(trevor): #580 redundant parse
-      this.parsedSupergraphSdl = parse(supergraphSdl);
-      this.supergraphSdl = supergraphSdl;
-      this.updateWithSchemaAndNotify(schema, supergraphSdl, true);
-    } catch (e) {
-      this.state = { phase: 'failed to load' };
-      throw e;
-    }
-    this.state = { phase: 'loaded' };
+  private getUplinkEndpoints(config: ManagedGatewayConfig) {
+    // 1. If config is set to a `string`, use it
```

Comment:
I know you just moved this around but I'm not sure what it means exactly

Reply:
fa7dc256